BioArchLinux/ibs/PKGBUILD

```diff
@@ -0,0 +1,21 @@
+# Maintainer: kiri <kiri@vern.cc>
+pkgname=ibs
+_pkgname=IBS
+pkgver=1.0.3
+pkgrel=1
+pkgdesc="Illustrator for Biology"
+arch=('any')
+url="https://ibs.renlab.org"
+license=('custom:commercial')
+depends=('java-environment=8' 'bash')
```

Comment:
Do you really need `java-environment`?  Have you read https://wiki.archlinux.org/title/Java_package_guidelines#Dependencies? I recommend to use `java-runtime`

Reply:
i totally agree with u! i will change now